docs/source/api/safe/threebody/soi.rst

```diff
@@ -0,0 +1,5 @@
+Patched Conics Module
```

Comment:
Can we change the name to "Sphere of influence module"?

Reply:
Okay!